googlecloud-plugins/widgets/BigQuery-batchsource.json

```diff
@@ -0,0 +1,48 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "BigQuery",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Project ID",
+          "name": "projectId"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "JSON Key File Path",
+          "name": "jsonFilePath"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Google Temporary Storage Path",
+          "name": "tempBucketPath"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Fully-qualified Input Table ID",
+          "name": "inputTableId"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Import Query",
+          "name": "importQuery"
+        },
+        {
+          "widget-type": "textbox",
```

Comment:
is this schema expected to be like other plugins' schema? if so this should follow other plugins widget for schema